mutiny-core/src/nodemanager.rs

```diff
@@ -940,8 +948,8 @@ impl<S: MutinyStorage> NodeManager<S> {
         };
 
         Ok(MutinyBip21RawMaterials {
-            address,
-            invoice: bolt11,
+            address: address.address,
+            invoice: Some(bolt11),
```

Comment:
should we just make this None if creating the invoice fails?

Reply:
No, there's specific errors that need to be returned so that paul can do the right thing or make the right warning.